src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator has stopped working.");
```

Comment:
Similarly, the stop message should be more specific. For instance, 'Excavator has stopped working.' could be improved to 'Excavator has finished its work.' to make it clear that the work cycle is complete.